packages/@azure/servicebus/data-plane/test/sessionsTests.spec.ts

```diff
@@ -182,8 +183,9 @@ describe("SessionTests - Accept a session by passing non-existing sessionId rece
       return Promise.resolve();
     }, unExpectedErrorHandler);
 
-    await delay(2000);
-    should.equal(receivedMsgs.length, 1);
+    const msgsCheck = await DelayStreaming(() => receivedMsgs.length === 1);
+    should.equal(msgsCheck, true, "Could not receive the messages in expected time.");
```

Comment:
There is another delay in line 173 that got missed

Reply:
I purposefully left that as we expect zero messages 
and `await DelayStreaming(() => receivedMsgs.length === 0);` won't even wait for 1 second.

Reply:
Never mind, we can keep it as is.